.gitignore

```diff
@@ -7,6 +7,7 @@
 .local
 .vscode
 .wasm-binaries
+*.adoc
```

Comment:
I would not ignore the adoc files but removing them as you did is great.

Most are very old. 

A (long) while ago, I introduced adoc to substrate mainly because it supports `include`. 

A point was made however back then: Rust means Markdown and there is no foreseen support for adoc so using adoc for the documentation of a Rust project adds some burden that it not worth going thru when we ned a simple README.

Reply:
I removed all adocs. The gitignore was meant as a warning for devs to not accidentially commit adocs again :smile: